python/loops-lists/lists.py

```diff
@@ -0,0 +1,29 @@
+# Python Course
+# Lists
+
+# make a list of names:
+names = ["Joe", "Mary", "Eugene", "Katrina"]
+
+# print the whole list:
+print("here is a list of names", names)
+
+# print the length of the list:
+print("the length of the list is", len(names))
+
+# print the first item:
+print("The first name in the list is", names[0])
+
+# print the second item:
+print("The second name in the list is", names[1])
+
+# remember how we "add" strings?
+x = "hello, " + "world"
+print(x)
+
+# we can do the same with lists
+y = [1, 2, 3] + [9, 4, 1]
+print(y)
```

Comment:
can you show list concat with a variable too.
```
cats = ["russian blue", "siamese", "burmese", "ragdoll"]
tabby = "tabby"
cats = cats + [tabby]
```
to avoid getting into `append()`?

Reply:
i know you have it later, but this seems appropriate in the basics